### DISCUSSION TITLE
Using WYSIWYG premium plugins (Tiny.cloud)

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#5039] [Archive]
Is there currently a way to use a premium subscription api-key (from tiny.cloud) with the WYSIWYG? We would like to use some of the premium plugins for compliance such as spell-checking, accessibility and link checking. 

I've looked into the WYSIWYG interface code and I believe the answer is "no." So on that note, if the interface were to be extended, what would be the Directus suggested method for handling a config option like an API-key for a given interface?

**Comment by rijkvanzanten** [comment]
> Is there currently a way to use a premium subscription api-key (from tiny.cloud) with the WYSIWYG?

No, the WYSIWYG interface relies on the "regular" non-cloud TinyMCE, which is incompatible with the Cloud offering. You'll have to set it up as a custom interface. 

> So on that note, if the interface were to be extended, what would be the Directus suggested method for handling a config option like an API-key for a given interface?

In this case, seeing it's a custom interface with a client-side token, it should be just fine to hardcode it in your interface 👍🏻
